kratos/tests/cpp_tests/sources/test_filesystem.cpp

```diff
@@ -15,8 +15,11 @@
 #include <fstream>
 
 // External includes
+#include "ghc/filesystem.hpp" // TODO after moving to C++17 this can be removed since the functions can be used directly
```

Comment:
In theory this is included in #include "includes/kratos_filesystem.h"

Reply:
removed :)